recipe/patches/001-remove-fancy-pypi-readme.patch

```diff
@@ -0,0 +1,67 @@
+From cf4a6c16634a20dda78aa08869496469e5d5d99a Mon Sep 17 00:00:00 2001
+From: Serhii Kupriienko
+Date: Fri, 22 Sep 2023 16:39:27 +0300
+Subject: [PATCH] Fix deps
+
+---
+ pyproject.toml                                | 37 +--------
+ 1 files changed, 1 insertions(+), 36 deletions(-)
+
+diff --git a/pyproject.toml b/pyproject.toml
+index fb8fae3..010357e 100644
+--- a/pyproject.toml
++++ b/pyproject.toml
+@@ -1,7 +1,7 @@
+ # SPDX-License-Identifier: MIT
+ 
+ [build-system]
+-requires = ["hatchling", "hatch-vcs", "hatch-fancy-pypi-readme"]
```

Comment:
instead of removing that dependency why we cannot add this as a dependent package ?

Reply:
It's not needed for building a package.  jsonschema and black packages have the same approach

Reply:
Indeed, `hatch-fancy-pypi-readme` is only required to generated the README in the generated wheel metadata (https://inspector.pypi.io/project/attrs/23.1.0/packages/f0/eb/fcb708c7bf5056045e9e98f62b93bd7467eb718b0202e7698eb11d66416c/attrs-23.1.0-py3-none-any.whl/attrs-23.1.0.dist-info/METADATA#line.53).